README.md

```diff
@@ -14,7 +14,7 @@
 
 ## Live Page
 
-[Page Link](https://kakalanp.github.io/TO-DO-list/)
+[Page Link](https://kakalanp.github.io/TO-DO-list/dist/)
 
 
 ## Getting Started
```

Comment:
- [ ] Let's document how to go about testing the project(commands to run).

Reply:
Added, thank you for the suggestion